src/theory/quantifiers/quantifiers_preprocess.h

```diff
@@ -0,0 +1,77 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Preprocessor for the theory of quantifiers.
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC5__THEORY__QUANTIFIERS__QUANTIFIERS_PREPROCESS_H
+#define CVC5__THEORY__QUANTIFIERS__QUANTIFIERS_PREPROCESS_H
+
+#include "proof/trust_node.h"
+#include "smt/env_obj.h"
+
+namespace cvc5 {
+namespace theory {
+namespace quantifiers {
+
+/**
+ * Module for doing preprocessing that is pertinent to quantifiers. These
+ * operations cannot be done in the rewriter since e.g. preskolemization
+ * depends on knowing the polarity of the position in which quantifiers occur.
+ */
+class QuantifiersPreprocess : protected EnvObj
+{
+ public:
+  QuantifiersPreprocess(Env& env);
+  /** preprocess
+   *
+   * This returns the result of applying simple quantifiers-specific
+   * preprocessing to n, including but not limited to:
```

Comment:
I think it'd be helpful to mention in the documentation all the transformations that are applied to the formula

Reply:
Isnt that what the document is already listing?

Reply:
I read "not limited to" as the listed ones being a strict subset of all the transformation applied.

Reply:
"not limited to" is so that this documentation is more maintainable: the description of this method should not be responsible for describing all of the methods it contains, those methods should be described more locally, e.g. in options help.

Reply:
Do we do that? The place where I'd look for what preprocessing is doing, apart from the implementation of this method itself, is the documentation in the method's signature

Reply:
Yes, for example: 
https://github.com/cvc5/cvc5/blob/master/src/theory/bags/bags_rewriter.h#L73

Reply:
I meant adding the list to options help or something. :) But regardless, This is a more general issue that should not block this PR. I'll approve.

Reply:
Oh, yeah, there are a lot of quantifiers options that should be better categorized, this is a larger TODO.